helm_deploy/demo-app/templates/deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: extensions/v1beta1
```

Comment:
The templates in this directory don't really seem to be Helm templates - they're just static YML. Is the Helm work being done in a separate branch or PR? If so we should remove the Helm stuff entirely. If not, it's best to create a skeleton chart with `helm create $chartname` and modify as required from there.

Reply:
So when I ran `helm create $chartname` I just overwrote the generated deployment files with the ones we already had to deployed through kubectl

Reply:
Correct me if I'm wrong but don't you specify the app version in the Chart.yaml?

Reply:
That’s the version of the chart itself rather than the app. Hem charts are just Go-templated YAML files, so if there’s no Go template stuff in there everything’s static any can’t be specified at install or upgrade time.

Reply:
Actually I misread your comment - the [`appVersion` field does indicate the version of the application](https://github.com/kubernetes/helm/blob/master/docs/charts.md#the-appversion-field), but by default it's just metadata that's displayed when searching or inspecting charts, and has no effect unless you make use of it in templated files.